pkl-core/src/test/files/LanguageSnippetTests/output/errors/invalidFileUri4.err

```diff
@@ -0,0 +1,13 @@
+–– Pkl Error ––
+Module URI `file:path/to/foo.pkl` has invalid syntax.
+
+x | res = import*("file:path/to/foo.pkl")
```

Comment:
Ideally, this should have the same error message as `invalidFileUri2`; the error is that the glob pattern is invalid.

But, this is somewhat challenging to fix, because the `URISyntaxException` gets thrown before `ImportGlobNode` is initialized.

Regardless, this error is good enough (given this error message, users know what to do).